README.md

```diff
@@ -39,7 +39,7 @@ Tests can be run using `rspec`
 
 Service can be run locally using `bundle exec rails server`
 
-### Running on Blazegraph
+### Running on Blazegraph - for OSX
```

Comment:
OK, changed to `macOS` after looking at the Apple-Logo -> About this Mac on my laptop running macOS Sierra.